### PR TITLE
fix(css): Update rgb hsl values as per the latest syntax

### PR DESCRIPTION
**Pull request by dipikabh**
Updates the [rgb()](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb) and a few [hsl()](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/hsl) values to follow the latest syntax:

- rgba -> rgb
- Commas are omitted.
- The alpha value is specified as a percentage separated by a slash (`/`).

UPDATE: PR to update prose/examples in content --> https://github.com/mdn/content/pull/30604

### CODE REVIEW

**Review by bsmth — APPROVED** [review]
Looks great, tnx. We might want to check in the content repo if we refer to any of these examples in this way:

>in the following example, we use `rgb(1,2,3)` or `rgba(1,2,3,4)`


__edit:__ indeed we need to update some places in content that reference the examples in prose.

Need update:

- [ ] Learn/CSS/CSS_layout/Positioning
- [ ] Learn/CSS/Building_blocks/Backgrounds_and_borders
- [ ] Learn/CSS/Building_blocks/Advanced_styling_effects
- [ ] Learn/CSS/CSS_layout/Media_queries 
  - (Walkthrough: a simple mobile-first layout) 

Look okay to me:

- [x] Learn/CSS/Building_blocks/The_box_model
- [x] Learn/CSS/Building_blocks/Debugging_CSS
- [x] Learn/CSS/CSS_layout/Floats_skills
- [x] Learn/CSS/CSS_layout/Grid_skills

**Comment by teoli2003** [comment]
I think that they are missing returns at the end of some of these files.

**Comment by bsmth** [comment]
> I think that they are missing returns at the end of some of these files.

yeah let's add in this PR. I will commit your suggestions, tnx

File: learn/inspecting/inspecting.html [hunk 1]
```diff
@@ -1,80 +1,82 @@
...
+    <p>
+      Turnip <em class="special">greens</em> yarrow ricebean rutabaga endive
+      cauliflower <em>sea lettuce</em> kohlrabi amaranth water spinach avocado
+      daikon napa cabbage asparagus winter purslane kale. Celery potato
+      scallion desert raisin horseradish spinach carrot soko.
+    </p>
+  </div>
+</body>
+
+</html>
```
Comment (bsmth) [inline learn/inspecting/inspecting.html:82]:
```suggestion
</html>

```

File: learn/media-queries/grid.html [hunk 2]
```diff
@@ -1,220 +1,216 @@
...
+      </aside>
+    </main>
+
+    <footer>
+      <p>&copy;2019</p>
+    </footer>
+  </div>
+</body>
+
+</html>
```
Comment (bsmth) [inline learn/media-queries/grid.html:216]:
```suggestion
</html>

```

File: learn/media-queries/step1.html [hunk 3]
```diff
@@ -1,148 +1,144 @@
...
+      <p>&copy;2019</p>
+    </footer>
+  </div>
+</body>
 
-      <footer><p>&copy;2019</p></footer>
-    </div>
-  </body>
-</html>
+</html>
```
Comment (bsmth) [inline learn/media-queries/step1.html:144]:
```suggestion
</html>

```

File: learn/media-queries/step2.html [hunk 4]
```diff
@@ -1,164 +1,160 @@
...
+      </aside>
+    </main>
+
+    <footer>
+      <p>&copy;2019</p>
+    </footer>
+  </div>
+</body>
+
+</html>
```
Comment (bsmth) [inline learn/media-queries/step2.html:160]:
```suggestion
</html>

```

**Comment by bsmth** [comment]
> I think that they are missing returns at the end of some of these files.

I've done the rest in https://github.com/mdn/css-examples/pull/158/commits/bfe1480f376c8a7068523f22887da4383c6f3006

**Comment by dipikabh** [comment]
Thanks for reviewing, @bsmth.

> edit: indeed we need to update some places in content that reference the examples in prose. 

Yeah, let me open a content PR alongside before merging this one

**Comment by dipikabh** [comment]
Updated the following:

- [ ] Learn/CSS/Building_blocks/Backgrounds_and_borders
- [ ] Learn/CSS/Building_blocks/Advanced_styling_effects
- [ ] Learn/CSS/Building_blocks/Values_and_units
- [ ] Learn/CSS/CSS_layout/Media_queries
- [ ] Learn/CSS/CSS_layout/Positioning

https://github.com/mdn/content/pull/30604

**Comment by dipikabh** [comment]
Merging this one now

**Pull request merged**